contributing.md

```diff
@@ -70,3 +72,26 @@
 ### Reference
 * [GitHub forking](https://gist.github.com/Chaser324/ce0505fbed06b947d962)  
 * [Git cherry-pick](https://git-scm.com/docs/git-cherry-pick)
+<br>
```

Comment:
We probably don't need a `<br>` here, I think leaving the line blank would give the same effect